performance_regression_test.py

```diff
@@ -105,17 +137,116 @@ def display_results(self, results):
         test_xml = ""
         for single_result in results:
             self.display_single_result(single_result)
-            test_xml += self.get_test_xml(single_result)
+            test_xml += self.get_test_xml(single_result, test_name=test_name)
 
         f = open(os.path.join(self.logdir,
                               'jenkins_perf_PerfPublisher.xml'), 'w')
-        content = """<report name="simple_regression_test report" categ="none">
+        content = """<report name="%s report" categ="none">
 %s
-</report>""" % test_xml
+</report>""" % (test_name, test_xml)
 
         f.write(content)
         f.close()
 
+    def add_stress_cmd_params(self, result, cmd):
+        # parsing stress command and return dict with params
+        cmd = cmd.strip()
+        cmd = cmd.strip().split('cassandra-stress')[1].strip()
+        if cmd.split(' ')[0] in ['read', 'write', 'mixed']:
+            section = 'cassandra-stress ' + cmd.split(' ')[0]
+            result[section] = {}
+            if cmd.startswith('no-warmup'):
+                result[section]['no-warmup'] = True
+
+            match = re.search('(cl\s?=\s?\w+)', cmd)
+            if match:
+                result[section]['cl'] = match.group(0).split('=')[1].strip()
+
+            match = re.search('(duration\s?=\s?\W+)', cmd)
+            if match:
+                result[section]['duration'] = match.group(0).split('=')[1].strip()
+
+            match = re.search('( n\s?=\s?\W+)', cmd)
+            if match:
+                result[section]['n'] = match.group(0).split('=')[1].strip()
+
```

Comment:
Parameters of cassandra-stress contain two part, command and option.

https://docs.datastax.com/en/cassandra/2.1/cassandra/tools/toolsCStress_t.html

How about save them to two subsections?   @roydahan 
```
subsection = 'command' or 'option'
result[section][subsection][k] = v
```